tasks/level-1/1.8.yml

```diff
@@ -7,7 +7,7 @@
   yum:
     name: "*"
     state: latest
-    exclude: nginx*,java*
+    exclude: nginx*,java*,jre*,jdk*
```

Comment:
maybe `*jre*`, and `*jre*`

Reply:
Actually unless I am missing something, I am more for not including `jre` to this exclusion list. The Runtime is where all the vulnerabilities are exploited in Java and I'd really like to stay up-to-date on this. 

Also, do we really need to install and run the JRE if we already have JDK on the system? Technically, our JDK should be the only executable running Java on the system and not the JVM. Please let me know if I am wrong about this or if I am missing something here.